### PR TITLE
test: docs manifest tests

### PR DESCRIPTION
**Pull request by ruben-arts**
Started with this to avoid docs typo's in the future. @Hofer-Julian planted this seed in my head and couldn't let it go.

### CODE REVIEW

File: crates/pixi_manifest/src/manifest.rs [hunk 1]
```diff
@@ -2142,4 +2143,22 @@ bar = "*"
...
     }
+
+    #[test]
+    fn test_docs_pixi_manifests() {
+        let location = "../../docs/source_files/pixi_tomls/*.toml";
+
+        // Check that the glob is not empty
+        assert!(glob(location).unwrap().count() > 0);
+
+        for entry in glob(location).unwrap() {
```
Comment (Hofer-Julian) [inline crates/pixi_manifest/src/manifest.rs:2154]:
```suggestion
        let entries = glob(location).unwrap();
        assert!(entries.count() > 0);

        for entry in entries {
```

File: crates/pixi_manifest/src/manifest.rs [hunk 2]
```diff
@@ -2142,4 +2143,22 @@ bar = "*"
...
+            match entry {
+                Ok(path) => {
+                    let contents = std::fs::read_to_string(path).unwrap();
+                    let _manifest = Manifest::from_str(Path::new("pixi.toml"), contents).unwrap();
```
Comment (Hofer-Julian) [inline crates/pixi_manifest/src/manifest.rs:2158]: Any reason why you don't use `from_path` here?

**Review by Hofer-Julian — APPROVED** [review]
Small nits, apart from that it looks good!

File: crates/pixi_manifest/src/manifest.rs (hunk 1 above)
Comment (ruben-arts) [reply crates/pixi_manifest/src/manifest.rs:2154]: I've been fighting with this one test for over 30 minutes.

The issues is that count() consumes entries. And i'm not allowed to clone() it. 

So this was the solutions.

But my real hope is that we could use #[files()] in combination with rstest but I wasn't able to get it working.

Comment (Hofer-Julian) [reply crates/pixi_manifest/src/manifest.rs:2154]: I see, then let's leave it as is :)

File: crates/pixi_manifest/src/manifest.rs (hunk 2 above)
Comment (Hofer-Julian) [reply crates/pixi_manifest/src/manifest.rs:2158]: Okay!

**Pull request merged**